cirq/contrib/quimb/mps_simulator.py

```diff
@@ -505,14 +531,13 @@ def estimation_stats(self):
 
         # The computation below is done for numerical stability, instead of directly using the
         # formula:
-        # estimated_fidelity = (1 - self.rsum2_cutoff) ** self.num_svd_splits
-        estimated_fidelity = 1.0 + np.expm1(np.log1p(-self.rsum2_cutoff) * self.num_svd_splits)
+        # estimated_fidelity = \prod_i (1 - e_n_s_i)
+        estimated_fidelity = 1.0 + np.expm1(sum([np.log1p(-x) for x in self.e_n_s]))
```

Comment:
`sum` can operate directly on list comprehensions, e.g. `sum(x for x in range(5))`

Reply:
Done.